### ISSUE TITLE
E121: Undefined variable: login (solution: use --netrc-file instead)

### ISSUE THREAD

**Issue by justinmk** [tpope/vim-rhubarb#11]
TL;DR: The curl `--netrc-file` option might be more robust than plain `--netrc`.

On Windows with curl 7.40 with `~/.netrc` configured correctly, the curl invocation:

    curl -q --silent -H 'Accept: application/json' -H 'Content-Type: application/json' -A rhubarb.vim --netrc 'https://api.github.com/repos/justinmk/config/collaborators'

returns this response:

    {"message":"Requires authentication","documentation_url":"https://developer.github.com/v3"}

Using `--netrc-file $HOME/.netrc` works, and allows rhubarb omni-completion to work.

---

A related UI issue is that the above authentication failure results in this user-facing message:

```
Error detected while processing function rhubarb#omnifunc:
line    7:
E121: Undefined variable: login
```
